dev-aws/kafka-shared-msk/cbc/cbc.tf

```diff
@@ -1301,7 +1299,6 @@ module "cbc_legacy_account_eqdb_loader" {
 module "cbc_account_events_relay" {
   source           = "../../../modules/tls-app"
   produce_topics   = [kafka_topic.legacy_account_events_v2.name]
-  consume_topics   = ["account-identity.public.account.events"]
   consume_groups   = ["account-identity.cbc-account-events-relay-v2"]
```

Comment:
I think it would be good to move also the related consumer group

Reply:
> I think it would be good to move also the related consumer group

is it not enough to have a single definition for the ACLs for those consumer groups (i.e. [here](https://github.com/utilitywarehouse/kafka-cluster-config/blob/d25341d70adae0880b323b7f83d666ea13f3357f/modules/tls-app/main.tf#L23)? Or would you want to duplicate them for visibility?

Reply:
>is it not enough to have a single definition for the ACLs for those consumer groups

It would be enough. 
But consuming always happens within a consumer group. So while we moved the consume_topics to the module containing the topic, it would be reasonable to move also the consumer groups used for consuming from that topic.